SampleUsages/TestSamples/Queue-CSharp-HighMemory/run.csx

```diff
@@ -4,9 +4,9 @@ public static void Run(string sizeMb, out string output, TraceWriter log)
 {
     var arrSize = int.Parse(sizeMb) * 1024 * 1024;
     var bytes = new byte[arrSize];
-    for (int i = 0; i < arrSize; i++)
+    for (int i = 0; i < arrSize; i += 256)
```

Comment:
A static array would mean that we don't have an allocation per message. That would make the memory usage constant, regardless of how many messages the given function app recieves. It would fail to create a difference between low/medium/high scenarios.

This scenario does need to be rethought (e.g. should we move to generating memory pressure that cannot be addressed through GC?), but for now my goal was just to get things back to 16mb per message and running at comparable scale to the CPU scenario.